App.tsx

```diff
@@ -11,14 +11,12 @@ LogBox.ignoreLogs([
 ]);
 
 export const App: FC = () => {
-  const [userData, setUserData] = useState(true);
+  const isLoggedIn = useSelector<AppStoreType, boolean>(
+    (state) => state.login.isLoggedIn,
+  );
```

Comment:
you can create a separate folder called `selectors` and create a separate selectors for each reducer